site/content/en/docs/tasks/troubleshooting/troubleshooting_provreq.md

```diff
@@ -0,0 +1,102 @@
+---
+title: "Troubleshooting Provisioning Request"
+date: 2024-05-20
+weight: 3
+description: >
+  Troubleshooting the status of a Provisioning Request
+---
+
+This doc is about troubleshooting ProvisioningRequests, an API defined by [Cluster Autoscaler](https://github.com/kubernetes/autoscaler/blob/4872bddce2bcc5b4a5f6a3d569111c11b8a2baf4/cluster-autoscaler/provisioningrequest/apis/autoscaling.x-k8s.io/v1beta1/types.go#L41).
+
+See the details about  Provisioning Request[here](https://cloud.google.com/kubernetes-engine/docs/how-to/provisioningrequest).
+
+Provisioning Requests are created in Kueue with [Provisioning Admission Check Controller](/docs/admission-check-controllers/provisioning/), and are treated by Kueue like an [Admission Check](/docs/concepts/admission_check/). It means Provisioning Requests needs to succeed for a Workload to be admitted.
+
+## Before you begin
+
+Before you begin further investigation make sure these fundamental requirements are met:
+- Your cluster's version is at least 1.28.3-gke.1098000 with the Cluster Autoscaler at least 28.122.0
+- Kueue's version is at least 0.5.0
+- You have enabled the `ProvisioningACC` in [the feature gates configuration](/docs/installation/#change-the-feature-gates-configuration)
+- Your node group enables it. It may vary depending on your cloud provider.
+
+## Provisioning Request's state
+
+You can run a following command to see a brief state of a Provisioning Request (and other Admission Checks) in the `Admission Checks` field of the Workload's Status.
+
+```bash
+kubectl describe workload WORKLOAD_NAME
+```
+
+Kueue adds the annotation `cluster-autoscaler.kubernetes.io/consume-provisioning-request` with Provisionig Request's name as a value, which allows identifying a corresponding Provisioning Request to your Workload. You can find this annotation in Workload's Status in the `Admission Checks` field.
+
+One of the reason your job is not running is that ProvisioningRequest is waiting to be provisioned. To find out if this is the case you can view Provisioning Request's state by running the following command:
```

Comment:
In the beginning of the paragraph you suggest to check state of a Provisioning Request running `kubectl describe workload` and here you suggest run `kubectl get provisioningrequest`. 

Probably you the `kubectl describe workload` is aimed to find if the ProvReq was created and what the name of it and see short info.

I would put this paragraph in the begening of the section, then info about annotation and then commands `describe workload`, `get provreq` etc.
Also, could  you add an example of output of the command?

Reply:
Thank you, please take a look if the changes align with your idea

Reply:
All good, thank you!